test/e2e/utils.go

```diff
@@ -153,9 +155,9 @@ func prepare(t *testing.T) (*framework.TestCtx, error) {
 	f := framework.Global
 	// wait for the operator to be ready
 	if !usingOLM {
-		// TODO replace 10*time.minute with timeout once #947 is fixed
-		err := e2eutil.WaitForDeployment(t, f.KubeClient, namespace, "jaeger-operator", 1, retryInterval, 10*time.Minute)
+		err := e2eutil.WaitForDeployment(t, f.KubeClient, namespace, "jaeger-operator", 1, retryInterval, timeout)
```

Comment:
Just wondering if this will cause issues, as the default timeout I believe is 2 mins, so alot less the 10?

Reply:
Yes, default is 2, but normally the operator should deploy in 10-15 seconds.  For 947 it was often taking more than 5 minutes, so we bumped it up to avoid that.